cli.js

```diff
@@ -30,11 +31,6 @@ const handleCommand = async function handleCommand(options, specificHandler) {
   if (realm) {
     forceRealm(realm);
   }
-  const cmdSeq = options._;
```

Comment:
This code gives an error in case we have an extra command, for example:
`bn list something-else`
without this code, `yargs` will just ignore it

this code relies on the fact that we don't have subcommands, which is not true now

Reply:
I understand now, thanks for explaining your rationale. That being said, I wrote the code for the problem you described. It seems like a bad user experience to not error on superfluous extra commands. Can you either

a) keep the existing functionality and add logic for special exceptions (as with triggers)
b) at minimum open an issue for a)

Reply:
Taking another look, `yargs` ignores `bn list something-else` with or without this code, without an error message
So I think it is broken anyway...